models/impossible_finance/bnb/impossible_finance_IDO_pools.sql

```diff
@@ -0,0 +1,104 @@
+{{
+    config(
+        alias='IDO_pools',
```

Comment:
please use lowercase on `IDO` for consistency across spellbook. this also applies to the file name itself.